client/dist/styles/product_overview.css

```diff
@@ -39,12 +39,15 @@
   margin: 0 auto;
 }
 
-.carousel_expand{
-  height: 15px;
-  width: 15px;
-  display: flex;
-  justify-content: flex-end;
-  cursor: pointer
+.expand_clicked {
```

Comment:
Consider more descriptive class names so it is easier to know what is being impacted by style.